scm/src/scm_input.F90

```diff
@@ -2259,8 +2318,8 @@ subroutine get_case_init_DEPHY(scm_state, scm_input)
     !set all individual w forcing controls to .true. until finer control is available from the input file
     scm_state%force_sub_for_T = .true.
     scm_state%force_sub_for_qv = .true.
-    scm_state%force_sub_for_u = .true.
-    scm_state%force_sub_for_v = .true.
+    !scm_state%force_sub_for_u = .true.
```

Comment:
This can potentially be uncommented. Subsidence forcing typically isn't applied to horizontal momentum and the DEPHY standard doesn't have a control for which variables to apply w or omega forcing to. This is simply an assertion on my part.